app/views/paths/index.html.erb

```diff
@@ -44,7 +44,7 @@
               <% elsif current_user.path === path %>
                 <%= button_to 'Resume', path_url(path), method: :get, class: 'button button--primary card-main__button' %>
               <% else %>
-                <%= button_to 'Switch', users_paths_path(path_id: path.id), class: 'button button--secondary card-main__button', remote: false, method: :post, data: { disable_with: 'Switching...' } %>
+                <%= button_to 'Switch', users_paths_path(path_id: path.id), class: 'button button--secondary card-main__button', remote: false, method: :post, data: { disable_with: 'Switching...', test_id: path.title } %>
```

Comment:
What do you think about appending `<path title>-switch-path-btn` to the test id for the switch path buttons? I think it might make it a little clearer whats being clicked in the test. Just thought id raise it since my first thought when seeing `find(:test_id, 'Rails').click` was we were clicking the path card for the rails course.

Reply:
@KevinMulhern Makes sense. Will sort